### DISCUSSION TITLE
Supported OS

### DISCUSSION THREAD

**Discussion by nirmalmahiya** [faaxm/spix#40] [Q&A]
I need to do test automation on Windows OS. Does the framework support Windows 10?

**Comment by faaxm** [comment]
Yes, Windows 10 should be supported. However, I personally use mostly linux and macOS, so I would recommend having a look and compiling the examples first.

Also, there seems to be an issue when using spix as dll (at least there is a PR open I still have to have a look at). Static linking will work fine though.

**Comment by vivek-vadher** [comment]
HI @nirmalmahiya and @faaxm 
Have you tried setup Spix in windows machine? I am trying to compile and build it on windows. But facing difficult with minGW compiler.
Also, I am not sure even after I compile the Spix successfully, how to use the .dll in Python Automation. Any response will be appreciated
